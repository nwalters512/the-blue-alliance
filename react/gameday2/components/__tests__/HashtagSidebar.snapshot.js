import React from 'react'
import renderer from 'react-test-renderer'
import HashtagSidebar from '../HashtagSidebar'

describe('HashtagSidebar snapshot', () => {
  beforeAll(() => {
    // componentDidMount tries to use functions on the document global,
    // which is unavailable outside the browser. Mock it out for
    // testing purposes.
    HashtagSidebar.prototype.componentDidMount = jest.genMockFunction()
  })

  it('renders correctly', () => {
    const component = renderer.create(
      <HashtagSidebar />
    ).toJSON()
    expect(component).toMatchSnapshot()
  })

  it('renders correctly when hidden', () => {
    const component = renderer.create(
      <HashtagSidebar hidden />
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
