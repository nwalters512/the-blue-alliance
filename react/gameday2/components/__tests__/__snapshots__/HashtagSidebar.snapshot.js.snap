exports[`HashtagSidebar snapshot renders correctly 1`] = `
<div
  className="hashtag-sidebar">
  <div
    id="twitter-widget">
    <a
      className="twitter-timeline"
      data-widget-id="406597120632709121"
      href="https://twitter.com/search?q=%23omgrobots">
      Tweets about \"#omgrobots\"
    </a>
  </div>
</div>
`;

exports[`HashtagSidebar snapshot renders correctly when hidden 1`] = `
<div
  className="hashtag-sidebar">
  <div
    id="twitter-widget">
    <a
      className="twitter-timeline"
      data-widget-id="406597120632709121"
      href="https://twitter.com/search?q=%23omgrobots">
      Tweets about \"#omgrobots\"
    </a>
  </div>
</div>
`;
